TOC Restructuring Final PR (#2402)
couchbase/docs-server: modules/fts/pages/fts-creating-index-from-UI-classic-editor.adoc

= Classic Editor

[abstract]
A classic editor is an advanced tool where users can directly configure the index mapping. 

It is intended for users already familiar with the concepts of full-text search.

To create an index, through the classic editor, left-click on the *Add Index* button towards the right-hand side.

The Add Index screen appears.

image::fts-add-index-initial.png[,400,align=left]

> To define a basic index on which Full Text Search can be performed, begin by entering a unique name for the index into the Name field, at the upper-left: for example, travel-sample-index. (Note that only alphanumeric characters, hyphens, and underscores are allowed for index names. Also note that the first character of the name must be alphabetic.) Then, use the pull-down menu provided for the Bucket field, at the upper-right, to specify the travel-sample bucket:

Reviewer: ```suggestion
To define a basic index on which Full Text Search can be performed, begin by entering a unique name for the index in the Name field, on the upper-left: for example, travel-sample-index. (Note that only alphanumeric characters, hyphens, and underscores are allowed for index names. Also note that the first character of the name must be alphabetic.) Then, use the pull-down menu provided for the Bucket field, on the upper-right, to specify the travel-sample bucket:
```

Author: updated
